Pipfile

```diff
@@ -10,7 +10,7 @@ pyjwt = "*"
 requests = "*"
 
 [dev-packages]
-black = "20.8b1"
+black = "==21.8b0"
```

Comment:
Are you just updating as there is a new version or  part of the bug?

Reply:
To fix the bug I had to update the packages and `20.8b1` of black is no longer usable, so I just used the latest one available.